### PR TITLE
Add Ubuntu 17.10, Artful

### PR DESCRIPTION
**Pull request by seemethere**
Signed-off-by: Eli Uriegas <eli.uriegas@docker.com>

### CODE REVIEW

**Comment by vielmetti** [comment]
I see armhf but not arm64 ?

File: deb/Makefile [hunk 1]
```diff
@@ -61,6 +61,19 @@ ubuntu-zesty: ## build ubuntu zesty deb packages
 		debbuild-$@/$(ARCH)
 	$(CHOWN) -R $(shell id -u):$(shell id -g) debbuild/$@
 
+.PHONY: ubuntu-artful
+ubuntu-artful: ## build ubuntu zesty deb packages
```
Comment (juliendufresne) [inline deb/Makefile]: In the comment: it's not zesty but artful :)

Comment (seemethere) [reply deb/Makefile]: Fixed! Thanks!

**Comment by the-kid89** [comment]
It would be nice to have docker on my system outside of a VM.

**Comment by tomekit** [comment]
I've simply installed old Ubuntu 17.04 deb from Docker official repositiories on my new Ubuntu 17.10 and it works fine to me.

Link: https://download.docker.com/linux/ubuntu/dists/xenial/pool/stable/amd64/
Download the recent file, in my case: docker-ce_17.09.0~ce-0~ubuntu_amd64.deb  

Run `sudo dpkg-i docker-ce_17.09.0~ce-0~ubuntu_amd64.deb `. Good luck

File: deb/ubuntu-artful/Dockerfile.x86_64 [hunk 2]
```diff
@@ -0,0 +1,28 @@
+FROM ubuntu:artful
+
+RUN apt-get update && apt-get install -y apparmor bash-completion btrfs-tools build-essential cmake curl ca-certificates debhelper dh-apparmor dh-systemd git libapparmor-dev libdevmapper-dev libltdl-dev libseccomp-dev pkg-config vim-common libsystemd-dev --no-install-recommends && rm -rf /var/lib/apt/lists/*
+
+ENV GO_VERSION 1.8.3
```
Comment (andrewhsu) [inline deb/ubuntu-artful/Dockerfile.x86_64]: bump to 1.8.4

File: deb/ubuntu-artful/Dockerfile.s390x [hunk 3]
```diff
@@ -0,0 +1,28 @@
+FROM s390x/ubuntu:artful
+
+RUN apt-get update && apt-get install -y apparmor bash-completion btrfs-tools build-essential cmake curl ca-certificates debhelper dh-apparmor dh-systemd git libapparmor-dev libdevmapper-dev libltdl-dev libseccomp-dev pkg-config vim-common libsystemd-dev --no-install-recommends && rm -rf /var/lib/apt/lists/*
+
+ENV GO_VERSION 1.8.3
```
Comment (andrewhsu) [inline deb/ubuntu-artful/Dockerfile.s390x]: bump to 1.8.4

File: deb/ubuntu-artful/Dockerfile.ppc64le [hunk 4]
```diff
@@ -0,0 +1,28 @@
+FROM ppc64le/ubuntu:artful
+
+RUN apt-get update && apt-get install -y apparmor bash-completion btrfs-tools build-essential cmake curl ca-certificates debhelper dh-apparmor dh-systemd git libapparmor-dev libdevmapper-dev pkg-config vim-common libseccomp-dev libsystemd-dev libltdl-dev --no-install-recommends && rm -rf /var/lib/apt/lists/*
+
+ENV GO_VERSION 1.8.3
```
Comment (andrewhsu) [inline deb/ubuntu-artful/Dockerfile.ppc64le]: bump to 1.8.4

File: deb/ubuntu-artful/Dockerfile.armv7l [hunk 5]
```diff
@@ -0,0 +1,28 @@
+FROM arm32v7/ubuntu:artful
+
+RUN apt-get update && apt-get install -y apparmor bash-completion btrfs-tools build-essential cmake curl ca-certificates debhelper dh-apparmor dh-systemd git libapparmor-dev libdevmapper-dev libltdl-dev libseccomp-dev pkg-config vim-common libsystemd-dev --no-install-recommends && rm -rf /var/lib/apt/lists/*
+
+ENV GO_VERSION 1.8.3
```
Comment (andrewhsu) [inline deb/ubuntu-artful/Dockerfile.armv7l]: bump to 1.8.4

File: deb/ubuntu-artful/Dockerfile.x86_64 (hunk 2 above)
Comment (andrewhsu) [reply deb/ubuntu-artful/Dockerfile.x86_64]: 1.8.5 now...golang moves fast

File: deb/ubuntu-artful/Dockerfile.s390x (hunk 3 above)
Comment (andrewhsu) [reply deb/ubuntu-artful/Dockerfile.s390x]: 1.8.5

File: deb/ubuntu-artful/Dockerfile.ppc64le (hunk 4 above)
Comment (andrewhsu) [reply deb/ubuntu-artful/Dockerfile.ppc64le]: 1.8.5

File: deb/ubuntu-artful/Dockerfile.armv7l (hunk 5 above)
Comment (andrewhsu) [reply deb/ubuntu-artful/Dockerfile.armv7l]: 1.8.5

**Pull request merged**